report_pdf_zip_download/static/src/js/action_manager_report.js

```diff
@@ -0,0 +1,88 @@
+// © 2017 Creu Blanca
+// Copyright 2024 Quartile (https://www.quartile.co)
+// License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+odoo.define("batch_report_pdf_zip.report", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var ActionManager = require("web.ActionManager");
+    var crash_manager = require("web.crash_manager");
+    var framework = require("web.framework");
+    var session = require("web.session");
+    var _t = core._t;
+
+    ActionManager.include({
+
+        _downloadReportZIP: function (url, actions) {
```

Comment:
I feel that there must be a better way to handle zip downloads, but I don't have a concrete suggestion...